recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "orange-widget-base" %}
-{% set version = "4.19.0" %}
+{% set version = "4.22.0" %}
```

Comment:
Isn't **v4.23.0** suitable too?

Reply:
No because it will conflict with the version of orange-canvas-core required for orange3.

Reply:
Unfortunately, it's not in this case because **v4.23.0** lists `orange-canvas-core>=0.2a.dev0,<0.3a` [here](https://github.com/biolab/orange-widget-base/blob/4.23.0/setup.py#L60), but that is incompatible with what `orange` needs [here](https://github.com/biolab/orange3/blob/3.36.2/requirements-gui.txt#L1)